### ISSUE TITLE
BUG: `gen-corim` is broken

### ISSUE THREAD

**Issue by thomas-fossati** [veraison/services#203]
### What version of the package are you using?

HEAD (at dfb068204473cad9c412337d5abef7ad88b8bc3b)

### Does this issue reproduce with the latest release?

yes

### What OS and CPU architecture are you using (`go env`)?

```sh
GO111MODULE=''
GOARCH='arm64'
GOBIN=''
GOCACHE='/Users/tho/Library/Caches/go-build'
GOENV='/Users/tho/Library/Application Support/go/env'
GOEXE=''
GOEXPERIMENT=''
GOFLAGS=''
GOHOSTARCH='arm64'
GOHOSTOS='darwin'
GOINSECURE=''
GOMODCACHE='/Users/tho/go/pkg/mod'
GONOPROXY=''
GONOSUMDB=''
GOOS='darwin'
GOPATH='/Users/tho/go'
GOPRIVATE=''
GOPROXY='https://proxy.golang.org,direct'
GOROOT='/usr/local/go'
GOSUMDB='sum.golang.org'
GOTMPDIR=''
GOTOOLCHAIN='auto'
GOTOOLDIR='/usr/local/go/pkg/tool/darwin_arm64'
GOVCS=''
GOVERSION='go1.21.0'
GCCGO='gccgo'
AR='ar'
CC='clang'
CXX='clang++'
CGO_ENABLED='1'
GOMOD='/Users/tho/Code/github.com/veraison/services/go.mod'
GOWORK=''
CGO_CFLAGS='-O2 -g'
CGO_CPPFLAGS=''
CGO_CXXFLAGS='-O2 -g'
CGO_FFLAGS='-O2 -g'
CGO_LDFLAGS='-O2 -g'
PKG_CONFIG='pkg-config'
GOGCCFLAGS='-fPIC -arch arm64 -pthread -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -ffile-prefix-map=/var/folders/st/1wblnvg95bg4p9hnzc6dwjzh0000gn/T/go-build2551571816=/tmp/go-build -gno-record-gcc-switches -fno-common'
```

### What did you do?

I tried to install the command following the instructions in the [README](https://github.com/veraison/services/blob/main/utils/gen-corim/README.md)

### What did you expect to see?

I expected the `gen-corim` command to be installed in my computer.

### What did you see instead?

First:

```sh
go: github.com/veraison/services/gen-corim@latest: module github.com/veraison/services@latest found (v0.0.0-20231006082242-dfb068204473), but does not contain package github.com/veraison/services/gen-corim
```

because the package path is not correct (it should be `github.com/veraison/services/utils/gen-corim@latest` instead).

Then, with the right path:

```sh
# github.com/veraison/services/utils/gen-corim/cmd
/Users/tho/go/pkg/mod/github.com/veraison/services@v0.0.0-20231006082242-dfb068204473/utils/gen-corim/cmd/gen-corim.go:390:53: undefined: comid.VerifKeys
/Users/tho/go/pkg/mod/github.com/veraison/services@v0.0.0-20231006082242-dfb068204473/utils/gen-corim/cmd/gen-corim.go:395:15: undefined: comid.NewVerifKey
/Users/tho/go/pkg/mod/github.com/veraison/services@v0.0.0-20231006082242-dfb068204473/utils/gen-corim/cmd/gen-corim.go:397:16: undefined: comid.NewVerifKeys
```

**Comment by thomas-fossati** [comment]
This bug was noted by @hannestschofenig

**Comment by yogeshbdeshpande** [comment]
@hannestschofenig 
PLease try https://github.com/veraison/gen-corim 
and report if still this is an issue

**Comment by yogeshbdeshpande** [comment]
`gen-corim` under veraison/services/utils is been deprecated!

**Comment by thomas-fossati** [comment]
> `gen-corim` under veraison/services/utils is been deprecated!

hmmm, deprecated?  Is that stated somewhere?

**Comment by thomas-fossati** [comment]
Fixed in #204
